get_restarts.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-#-----------------------------------------------------------
```

Comment:
Please update the readme with the submission details for doing this job on a compute node (line 34).

Reply:
Thank you @ClaraDraper-NOAA.

Reply:
Updated. Please take a look.